plugin-enclave-gradle/src/main/resources/com/r3/conclave/plugin/enclave/gradle/gramine/gramine-seccomp.json

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
Is this the same file as the one in the root of the repo? If so then just use that and copy it into the conclave-host.jar. There's no need to put it in the gramine bundle.